app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +15,32 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-taxon {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
+          border-right: 6px solid #0b0c0c;
+          border-top: 5px solid transparent;
+          content: '';
+          display: block;
+          height: 0;
+          left: 0;
+          margin-top: -6px;
```

Comment:
Please check this in Firefox - we saw it misbehaving with the breadcrumb links when I introduced negative margin on those.